Code4SocialGood/c4sg-web
# Project Detail Page - Display Image #88

**To test:**

1. Navigate in front end to `/project/view/2`, 'Upload Photo' button should be removed. 
2. Upload an image from back end using swagger UI at `/swagger-ui.html#!/project/uploadImageUsingPOST` or `POST /api/projects/{id}/image` with an id of `2`. Use the image below if necessary. 
![project_convoforgood](https://github.com/Code4SocialGood/c4sg-web/raw/master/src/assets/project/Project_ConvoForGood.jpg)

3. Navigate in front end to `/project/view/2`, uploaded image should be visible.
